frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -992,7 +992,7 @@ export default function SurveyEdit(): JSX.Element {
                                             {surveysEventsAvailable && (
                                                 <LemonField.Pure
                                                     label="User sends events"
-                                                    info="Note that these events are only observed, and activate this survey, in the current user session."
+                                                    info="Note that these events are only observed, and activate this survey, in the current user session. And it's only valid for events captured with the PostHog SDK, with the posthog.capture() function."
```

Comment:
```suggestion
                                                    info="Note that these events are only observed, and activate this survey, in the current user session for events captured with the PostHog SDK"
```

`posthog.capture()` is actually confusing since there are other methods that captures events such as alias, identify, autocapture, etc, they internally call capture but the user does not know it

maybe better?

Reply:
makes sense, liked your suggestion too

Reply:
@lucasheriques you resolved the comment but didnt commit my changes, was that intentional?